构建 Typst 文档时 apt fetch ghostscript 异常

- [ ] 我正在着手修复这个问题

https://github.com/OI-wiki/OI-wiki/actions/workflows/build-pdf-typst.yml

目前至少影响 3 个 PR。

----

部分 log 信息

> ❌ **OI-wiki Typst Build**: Process completed with exit code 100.
> ```pwsh
> E: Failed to fetch mirror+file:/etc/apt/apt-mirrors.txt/pool/main/g/ghostscript/libgs-common_10.02.1%7edfsg1-0ubuntu7.4_all.deb  404  Not Found [IP: 40.81.13.82 80]
> Fetched 239 MB in 19s (12.8 MB/s)
> E: Failed to fetch mirror+file:/etc/apt/apt-mirrors.txt/pool/main/g/ghostscript/libgs10-common_10.02.1%7edfsg1-0ubuntu7.4_all.deb  404  Not Found [IP: 40.81.13.82 80]
> E: Failed to fetch mirror+file:/etc/apt/apt-mirrors.txt/pool/main/g/ghostscript/libgs10_10.02.1%7edfsg1-0ubuntu7.4_amd64.deb  404  Not Found [IP: 40.81.13.82 80]
> E: Failed to fetch mirror+file:/etc/apt/apt-mirrors.txt/pool/main/g/ghostscript/ghostscript_10.02.1%7edfsg1-0ubuntu7.4_amd64.deb  404  Not Found [IP: 40.81.13.82 80]
> E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
> Error: Process completed with exit code 100.
> ```

--- a/.github/workflows/build-pdf-typst.yml
+++ b/.github/workflows/build-pdf-typst.yml
@@ -28,6 +28,7 @@ jobs:
         env:
           GH_TOKEN: ${{ github.token }}
         run: |
+          sudo apt-get update
           sudo apt-get install curl git unzip imagemagick libwebp-dev librsvg2-dev fonts-noto-cjk fonts-noto-cjk-extra
           sudo apt-get remove fonts-noto-color-emoji
 

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -40,7 +40,9 @@ jobs:
           pipenv install
       - name: Install C++
         if: steps.changed-files.outputs.any_changed == 'true'
-        run: sudo apt-get install -y g++
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y g++
       - name: Make the File list
         if: steps.changed-files.outputs.any_changed == 'true'
         run: echo "${{ steps.changed-files.outputs.all_changed_files }}" > res.txt
@@ -67,7 +69,9 @@ jobs:
         with:
           fetch-depth: 0
       - name: Install g++-9
-        run: apt-get install -y g++-9
+        run: |
+          apt-get update
+          apt-get install -y g++-9
       - name: Undefined Behaviors Check
         env: 
           FILES_TO_TEST: ${{ needs.correctness.outputs.files_to_test }}

